service/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM openjdk:8-jre
+
+COPY build/bootScripts/spring-boot-template /opt/app/bin/
+
+COPY build/libs/spring-boot-template.jar /opt/app/lib/
```

Comment:
After renaming build artefact we need to update Dockerfile (3 lines in the file)

Reply:
Still pending, `docker-compose build` will fail.